config/config/wc-config.yaml

```diff
@@ -175,6 +175,14 @@ falco:
     affinity: {}
     nodeSelector: {}
 
+  falcoExporter:
+    resources: {}
+    tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
```

Comment:
I guess we would like falco exporter to run on masters as well?

Reply:
Okay. I took these values from the chart values. But i can remove them if we want to.

Reply:
Sorry, it's me that is stupid!
Tolerations makes it **possible** for them to be scheduled on masters.